Assets/Scripts/Accelerometer.cs

```diff
@@ -1,35 +1,45 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 
 public class Accelerometer : MonoBehaviour
 {
+    private Vector2 _velocity;
     public Vector2 velocity
     {
         get { return _velocity; }
-        private set { _velocity = value; }
     }
-    private Vector2 _velocity;
+
+    private Vector2 _impactForce;
+    public Vector2 impactForce
```

Comment:
This should be initialized in Awake or Start

Reply:
Done: https://github.com/cjh15hub/terribletweeters/pull/1/commits/bf5906843a21d2e978aa0e82765b2f23f2a3315e#diff-b9a61da49fb7ea93aad54b4b02c2a551caa0e1f74721482088ba4ee4d1a269e6R22

Reply:
Done: https://github.com/cjh15hub/terribletweeters/commit/bf5906843a21d2e978aa0e82765b2f23f2a3315e#diff-b9a61da49fb7ea93aad54b4b02c2a551caa0e1f74721482088ba4ee4d1a269e6R22